lib/model/model.go

```diff
@@ -128,6 +126,9 @@ var (
 	errFolderNotRunning  = errors.New("folder is not running")
 	errFolderMissing     = errors.New("no such folder")
 	errNetworkNotAllowed = errors.New("network not allowed")
+	// errors about why a connection is closed
+	errIgnoredFolderRemoved = errors.New("folder no longer ignored")
+	errReplacingConnection  = errors.New("replacing connection")
```

Comment:
I thought (back in the day), the protocol had error codes to signify the reason in a cross implementation compatible way. Is that no longer the case?

Reply:
Nope, human readable string according to the docsé

> Close
>
> [...]
>
> Fields
> 
> The reason field contains a human readable description of the error condition.
>

Reply:
Ah, it's response error code.